testing/dart/image_filter_test.dart

```diff
@@ -297,4 +298,60 @@ void main() {
       ),
     );
   });
+
+  // Tests that FilterQuality.<value> produces the expected golden file.
+  group('ImageFilter|FilterQuality', () async {
+    final ImageComparer comparer = await ImageComparer.create();
+
+    /// Load the image of 'Dash in a Noogler hat'.
+    Future<Image> dashInNooglerHat() async {
+      final ImmutableBuffer buffer = await ImmutableBuffer.fromAsset('DashInNooglerHat.jpg');
+      final ImageDescriptor descriptor = await ImageDescriptor.encoded(buffer);
+      final Codec codec = await descriptor.instantiateCodec(targetWidth: 100, targetHeight: 100);
+      final FrameInfo frame = await codec.getNextFrame();
+      return frame.image;
+    }
+
+    /// Return the [image] magnified by a factor of 5.
+    Future<Image> scaleImage5x(Image image, FilterQuality quality) async {
+      final ImageFilter filter = ImageFilter.matrix(Float64List.fromList(<double>[
```

Comment:
I wouldn't necessarily combine the image filter w/ filter quality, since I think it will make it harder to tell the difference? You could just use a drawImage call, the FilterQuality on the paint is used in that case

Reply:
Yeah that's fair, definitely the intent is to test as little as possible. Let me make that change.